api/openapi.gen.json

```diff
@@ -1476,11 +1476,11 @@
     },
     "/pubkeys/{ID}": {
       "delete": {
-        "description": "A pubkey represents an SSH public portion of a key pair with name and body. If a pubkey was uploaded to one or more clouds, the deletion request will attempt to delete those SSH keys from all clouds. This means in order to delete a pubkey the account must have valid credentials to all cloud accounts the pubkey was uploaded to, otherwise the delete operation will fail and the pubkey will not be deleted from Provisioning database. This operation returns no body.\n",
+        "description": "Deletes SSH keys that were uploaded with the specified public key from all the clouds.\n",
```

Comment:
It deletes the key stored in our database AND deletes the uploaded ones, would it be clear to users from this? :thinking: 

Also we've removed the information about the possible failure, is that intentional?

Reply:
next commit fixes this issue. Information added back :)